commits_test.go

```diff
@@ -2,11 +2,60 @@ package gitlab
 
 import (
 	"fmt"
+	"io"
 	"net/http"
+	"os"
 	"reflect"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
+func TestGetCommit(t *testing.T) {
+	mux, server, client := setup()
+	defer teardown(server)
+
+	mux.HandleFunc("/api/v4/projects/1/repository/commits/b0b3a907f41409829b307a28b82fdbd552ee5a27", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+
+		f, err := os.Open("testdata/get_commit.json")
```

Comment:
I think this cleans up the tests, but this would be the first test case using it. Let me know what you think!

Reply:
Maybe we should create a small helper (in `gitlab_test.go`) that takes a file path and returns the content of the file? But I like the approach and all the big JSON strings in the tests don't make it very nice to read 👍